config.json

```diff
@@ -252,9 +252,14 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 3,
       "slug": "crypto-square",
-      "topics": []
+      "topics": [
+	"Filtering",
+	"Strings",
+	"Text Formatting",
+	"Transforming"
```

Comment:
These could do with being indented consistently with the rest of the config.json file.